src/domain/hanke/edit/HankeForm.tsx

```diff
@@ -252,22 +253,58 @@ const HankeForm: React.FC<React.PropsWithChildren<Props>> = ({
     [],
   ];
 
+  function mapToErrorListItem(error: ValidationError) {
```

Comment:
This function has some complex logic, maybe it should have unit tests?

Reply:
Added unit tests.